src/Nethermind/Nethermind.Core/Transaction.cs

```diff
@@ -348,12 +348,24 @@ public interface ITransactionSizeCalculator
     /// <summary>
     /// Holds network form fields for <see cref="TxType.Blob" /> transactions
     /// </summary>
-    public class ShardBlobNetworkWrapper(byte[][] blobs, byte[][] commitments, byte[][] proofs, ProofVersion version)
+    public class ShardBlobNetworkWrapper(byte[] flatBlobs, byte[] flatCommitments, byte[] flatProofs, ProofVersion version)
```

Comment:
would be great if those arrays could be pooled, but it may be to complicated to do properly.